pandas/tests/tseries/test_offsets.py

```diff
@@ -4899,3 +4900,27 @@ def test_all_offset_classes(self):
             first = Timestamp(test_values[0], tz='US/Eastern') + offset()
             second = Timestamp(test_values[1], tz='US/Eastern')
             assert first == second
+
+
+# ---------------------------------------------------------------------
+
+offset_classes = [getattr(offsets, x) for x in dir(offsets)]
```

Comment:
we already list the classes somewhere in this module iirc.

Reply:
Looks like a fixture defined in conftest, but only used in test_offsets.  Instead of checking for the correct types it assume that `offsets.__all__` is exactly what it needs.  Mind if I a) put it directly test_offsets and b) make it check explicitly to avoid future footgunning?